CONTRIBUTING.md

```diff
@@ -48,13 +54,13 @@ Files are automatically formatted with prettier.
 ## Release
 
 ```sh
-npm run release
+yarn release
```

Comment:
does this work now? I remember this throwing annoying errors earlier

Reply:
Last time I did it, it worked (latest yarn)